ndcube/tests/test_global_coords.py

```diff
@@ -0,0 +1,85 @@
+import astropy.units as u
+import pytest
+
+from ndcube.global_coords import GlobalCoords
+
+
+@pytest.fixture
+def global_coords(ndcube_3d_ln_lt_l):
+    return GlobalCoords(ndcube_3d_ln_lt_l)
+
+
+def test_adding_global_coords(global_coords):
+    coord1 = 1 * u.m
+    coord2 = 2 * u.s
+    global_coords.add('name1', 'physical_type1', coord1)
+    global_coords.add('name2', 'physical_type2', coord2)
+    assert global_coords.names == ('name1', 'name2')
+    assert global_coords.physical_types == ('physical_type1', 'physical_type2')
```

Comment:
Also check the value of the coord here.

Reply:
Changing from variable `names` to `values` for `keys`...

Reply:
Done